pkg/config/config.go

```diff
@@ -447,6 +449,17 @@ func BrowserFlags() []cli.Flag {
 				return nil
 			},
 		},
+		&cli.IntFlag{
+			Name:  "browser.readiness.wait-for-n-cycles",
```

Comment:
i'd name this something more like `wait-for-n-query-cycles` as it only affects queries